lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -766,13 +766,10 @@ public function get_item_schema() {
 	 */
 	public function get_collection_params() {
 		$query_params = parent::get_collection_params();
-		$query_params['context'] = array(
-			'default'            => 'view',
-			'description'        => 'Change the response format based on request context.',
-			'enum'               => array( 'embed', 'view', 'edit' ),
-			'sanitize_callback'  => 'sanitize_key',
-			'type'               => 'string',
-		);
+
+		$params['context']['default'] = 'view';
+		$params['context']['enum'] = array( 'embed', 'view', 'edit' );
```

Comment:
Collection params are not json schema, so I don't think this `enum` will do anything, so far as I can tell